pkg/agent/apiserver/handlers/podinterface/handler.go

```diff
@@ -72,3 +74,14 @@ func HandleFunc(aq monitor.AgentQuerier) http.HandlerFunc {
 		}
 	}
 }
+
+var _ common.TableOutput = new(Response)
```

Comment:
Could you do it for controllerinfo and agentinfo as well? thanks.

Reply:
Done.

Reply:
Done.